src/main/java/frc/robot/Constants.java

```diff
@@ -13,7 +20,106 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+  /** Controller on module speed for rotating to target Input degrees, output [0,1] */
+  public static final PIDController ROTATE_TO_TARGET_PID_CONTROLLER = new PIDController(0.030, 0, 0.000);
+  public static final double ROTATE_TO_SHOOT_FF = 0.1;
+
+  public static final double PLACEHOLDER_DOUBLE = 1.0;
+  public static final int PLACEHOLDER_INT = 1;
+
   public static class OperatorConstants {
-    public static final int kDriverControllerPort = 0;
+    public static final int DRIVER_CONTROLLER_PORT = 0;
+  }
+
+  public static final class ModuleConstants {
+    /**The MAXSwerve module can be configured with one of three pinion gears: 12T, 13T, or 14T.
+     * This changes the drive speed of the module (a pinion gear with more teeth will result in a
+     * robot that drives faster).*/
+    public static final int DRIVING_MOTOR_PINION_TEETH = 14;
+
+    /**Invert the turning encoder, since the output shaft rotates in the opposite direction of
+     * the steering motor in the MAXSwerve Module.*/
+    public static final boolean TURNING_ENCODER_INVERTED = true;
+
+    /**Calculations required for driving motor conversion factors and feed forward*/
+    public static final double DRIVING_MOTOR_FREE_SPEED_RPS = NeoMotorConstants.FREE_SPEED_RPM / 60;
+    public static final double WHEEL_DIAMETER_METERS = 0.0762;
+    public static final double WHEEL_CIRCUMFERENCE_METERS = WHEEL_DIAMETER_METERS * Math.PI;
+    /**45 teeth on the wheel's bevel gear, 22 teeth on the first-stage spur gear, 15 teeth on the bevel pinion*/
+    public static final double DRIVING_MOTOR_REDUCTION = (45.0 * 22) / (DRIVING_MOTOR_PINION_TEETH * 15);
```

Comment:
Can we go through and PLACEHOLDER all the things we're not sure about?
Some of the ones I've spotted:
DRIVING_MOTOR_PINION_TEETH 
DRIVING_MOTOR_REDUCTION 
WHEEL_DIAMETER_METERS (maybe we know this? but should double check and also use `Units.inchesToMeters` to make it clearer)
TRACK_WIDTH 
WHEEL_BASE


If we took the PID constants from last year then we can keep those for now